DesignSafe-CI/portal
# task/DES-2616: Add TAS institution to user profile and update it on login

## Overview: ##
Part of the V3 migration will involve synchronizing the data model across projects/publications and reducing the number of external API calls needed to fetch project details. This PR adds institutions to the user profile model, which will allow us to keep track of all the user information we need inside of projects without having to call TAS for each user.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2616](https://jira.tacc.utexas.edu/browse/DES-2616)

## Summary of Changes: ##

## Testing Steps: ##
1. Once the branch is pulled, make sure workers are restarted to pick up the changes (`docker restart des_workers`)
2. Log into the portal
3. in the django shell: 
```
from django.contrib.auth import get_user_model
user = get_user_model().objects.get(username=<your username>)
user.profile.institution
```
You should see your institution populated here.
